### PR TITLE
[SQL] Change default value of Inserting in table mri_upload

### PR DESCRIPTION
**Pull request by nicolasbrossard**
This changes the default value of column Inserting in table mri_upload from '0' to NULL. If the default value is '0', it is going to cause a bug in the initial status displayed in the mri_upload result table when a file is initially uploaded (fix for Redmine ticket #13833)

### CODE REVIEW

**Comment by MounaSafiHarab** [comment]
@nicolasbrossard 

just referencing the original pull request where existing projects had this change done properly but not the schema (or new installs):
https://github.com/aces/Loris/pull/1675/files

File: SQL/0000-00-00-schema.sql [hunk 1]
```diff
@@ -711,7 +711,7 @@ CREATE TABLE `mri_upload` (
   `UploadLocation` varchar(255) NOT NULL DEFAULT '',
   `DecompressedLocation` varchar(255) NOT NULL DEFAULT '',
   `InsertionComplete` tinyint(1) NOT NULL DEFAULT '0',
-  `Inserting` tinyint(1) NOT NULL DEFAULT '0',
+  `Inserting` tinyint(1) NOT NULL DEFAULT NULL,
```
Comment (MounaSafiHarab) [inline SQL/0000-00-00-schema.sql]: is it normal to have NOT NULL then default NULL?

File: modules/brainbrowser/php/brainbrowser.class.inc [hunk 2]
```diff
@@ -22,18 +22,8 @@ namespace LORIS\brainbrowser;
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-class Brainbrowser extends \NDB_Form
+class Brainbrowser extends \NDB_Page
```
Comment (MounaSafiHarab) [inline modules/brainbrowser/php/brainbrowser.class.inc]: not sure why this file is in the pull request?

Comment (nicolasbrossard) [reply modules/brainbrowser/php/brainbrowser.class.inc]: Huh? I am surprised to see that too...I didn't edit or commit this file. My commit seems to have "herited" changes that were done by Xavier on this file.

File: SQL/0000-00-00-schema.sql (hunk 1 above)
Comment (nicolasbrossard) [reply SQL/0000-00-00-schema.sql]: Very wrong actually!  :) I corrected this in another commit (see below)

File: SQL/0000-00-00-schema.sql [hunk 3]
```diff
@@ -711,7 +711,7 @@ CREATE TABLE `mri_upload` (
   `UploadLocation` varchar(255) NOT NULL DEFAULT '',
   `DecompressedLocation` varchar(255) NOT NULL DEFAULT '',
   `InsertionComplete` tinyint(1) NOT NULL DEFAULT '0',
-  `Inserting` tinyint(1) NOT NULL DEFAULT '0',
+  `Inserting` tinyint(1) DEFAULT NULL,
```
Comment (driusan) [inline SQL/0000-00-00-schema.sql]: @MounaSafiHarab What does the "Inserting" column signify? If it's whether or not it's currently being inserted (which is what I would assume) it seems like not null default false is the right default..

Comment (MounaSafiHarab) [reply SQL/0000-00-00-schema.sql]: we do distinguish between NULL (for a new upload), inserting=1 meaning the pipeline is in progress, and inserting=0, meaning the insertion completed.
This distinction is needed for the 'Progress' column in Imaging Uploader.

Using InsertionComplete in conjunction with Inserting could potentially be used, but is not...

**Pull request closed without merging**